### PR TITLE
dlrs: updates README and releasenotes.md

### PR DESCRIPTION
**Pull request by DnPlas**
1. README.md: adds supported Tensorflow versions
2. clearlinux/releasenotes.md:
  - Updates supported versions for Tensorflow, PyTorch, Clearlinux
  - Updates stacks components
  - Adds ML Compiler stack
  - Updates Multi-node section with link to instructions to
    PyTorch Jobs/TFJobs

Signed-off-by: Daniela Plascencia <daniela.plascencia@linux.intel.com>

### CODE REVIEW

File: dlrs/README.md [hunk 1]
```diff
@@ -10,9 +10,8 @@ The Deep Learning Reference Stack, is an integrated, highly-performant open sour
 
 This open source community release is part of an effort to ensure AI developers have easy access to all features and functionality of Intel platforms.To offer more flexibility, there are multiple versions of the Deep Learning Reference Stack:
 
-* Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI)  
-* TensorFlow with Eigen optimized for Intel Architecture  
-* TensorFlow with Intel® Deep Neural Networks Library (Intel® DNNL) primitives
+* Tensorflow 2.0 and 1.15 with Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI)
```
Comment (unrahul) [inline dlrs/README.md:13]: Let's add openvino and transformers info also here: 
- [OpenVINO™ - Deep Learning Deployment Toolkit v2019_R3](https://github.com/opencv/dldt/releases/tag/2019_R3)
- [Transformers - State-of-the-art Natural Language Processing for TensorFlow 2.0](https://github.com/huggingface/transformers)

File: dlrs/README.md [hunk 2]
```diff
@@ -10,9 +10,8 @@ The Deep Learning Reference Stack, is an integrated, highly-performant open sour
 
 This open source community release is part of an effort to ensure AI developers have easy access to all features and functionality of Intel platforms.To offer more flexibility, there are multiple versions of the Deep Learning Reference Stack:
 
-* Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI)  
-* TensorFlow with Eigen optimized for Intel Architecture  
-* TensorFlow with Intel® Deep Neural Networks Library (Intel® DNNL) primitives
+* Tensorflow 2.0 and 1.15 with Intel® Deep Neural Networks Library (Intel® DNNL) primitives and AVX-512 Deep Learning Boost (formerly known as AVX-512 VNNI) 
+* TensorFlow with Eigen optimized for Intel Architecture
 * OSS PyTorch DLRS Docker image with Eigen
 * PyTorch DLRS Docker image with Intel® Math Kernel Library and Intel® Deep Neural Networks Library (Intel® DNNL)
```
Comment (unrahul) [inline dlrs/README.md]: - [Fliar -  A very simple framework for state-of-the-art Natural Language Processing](https://github.com/zalandoresearch/flair)
- [PyTorch-lightning - PyTorch wrapper for ML researchers](https://github.com/williamFalcon/pytorch-lightning)

File: dlrs/clearlinux/releasenote.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+The release includes:
+ * TVM 0.6
+
+## How to get the Deep Learning Reference Stack
+
+The official Deep Learning Reference Stack Docker images are hosted at: https://hub.docker.com/u/clearlinux/.  Note that the Intel MKL-DNN-VNNI version is also referred to as the Intel® MKL with AVX-512 Deep Learning Boost in the documentation. 
+
+ * Pull from the [Eigen version](https://hub.docker.com/r/clearlinux/stacks-dlrs-oss/)
+ * Pull from the [Tensroflow 1.15 and Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks-dlrs-mkl/)
```
Comment (unrahul) [inline dlrs/clearlinux/releasenote.md:64]: Let's have the MKL versions first (1.15, 2.0 and then Eigen)

File: dlrs/clearlinux/releasenote.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+ * Pull from the [Tensorflow 2.0 and Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks-dlrs_2-mkl/)
+ * Pull from the [PyTorch with OpenBLAS version](https://hub.docker.com/r/clearlinux/stacks-pytorch-oss)
+ * Pull from the [PyTorch with Intel MKL-DNN version](https://hub.docker.com/r/clearlinux/stacks-pytorch-mkl)
```
Comment (unrahul) [inline dlrs/clearlinux/releasenote.md:67]: Same here, Intel MKL-DNN version above openblas version

**Review by unrahul — COMMENTED** [review]
Please see my commnets, otherwise looks good to me, thank you @DnPlas

File: dlrs/README.md (hunk 1 above)
Comment (DnPlas) [reply dlrs/README.md:13]: Agree on OpenVINO, but perhaps transformers are better on the releasenotes.md file?

File: dlrs/README.md (hunk 2 above)
Comment (DnPlas) [reply dlrs/README.md]: Better have these on releasenotes.md?

File: dlrs/clearlinux/releasenote.md (hunk 3 above)
Comment (DnPlas) [reply dlrs/clearlinux/releasenote.md:64]: +1

File: dlrs/clearlinux/releasenote.md (hunk 4 above)
Comment (DnPlas) [reply dlrs/clearlinux/releasenote.md:67]: +1

File: dlrs/README.md (hunk 1 above)
Comment (unrahul) [reply dlrs/README.md:13]: +1 , yeah that makes more sense.

File: dlrs/README.md (hunk 2 above)
Comment (unrahul) [reply dlrs/README.md]: +1

**Pull request merged**